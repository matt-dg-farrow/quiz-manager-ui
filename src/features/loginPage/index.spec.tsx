import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import LoginPage from ".";

describe("The Header component", () => {
  const wrapper = shallow(<LoginPage />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
