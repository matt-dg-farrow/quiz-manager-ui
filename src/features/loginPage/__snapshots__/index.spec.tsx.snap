// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The LoginPage component should contain the correct number of components 1`] = `
<div
  className="page-content-container"
>
  <PageTitle
    cssClass="quiz-manager-page-title-login"
    text="Welcome to the WebbiSkools Quiz Manager!"
  />
  <div
    className="login-page-item"
  >
    <Label
      cssClass="login-page-label"
      name="username"
    />
    <TextInput
      name="username"
    />
  </div>
  <div
    className="login-page-item"
  >
    <Label
      cssClass="login-page-label"
      name="password"
    />
    <TextInput
      name="password"
    />
  </div>
  <Link
    className="quiz-manager-button-link"
    to="/quiz"
  >
    Login
  </Link>
</div>
`;
